src/app/components/Input/Input.tsx

```diff
@@ -6,10 +6,10 @@ import { useFormField } from "../Form/useFormField";
 type InputProps = { as?: React.ElementType; isInvalid?: boolean } & React.HTMLProps<any>;
 
 const InputStyled = styled.input`
-	${tw`py-3 px-4`}
+	${tw`px-4 py-3`}
```

Comment:
It was broken by this line, having the padding in the `styled` does not allow it to be override, should be moved to `className`

Reply:
Yeah, make sense! Updating... :oleglala: